cypress/integration/singUp.spec.js

```diff
@@ -0,0 +1,63 @@
+describe('Sing Up', () => {
+  beforeEach( () => {
+   cy.fixture('user.json').as('userData')
+   cy.visit('baseUrl')
+   cy.get('.nav-link').eq(2).click()
+  })
+  it('Format email incorrect', () => {
+    cy.get('@userData').then((userData) => {
+     cy.get('.form-control').eq(0).type(userData.username)
+     cy.get('.form-control').eq(1).type(userData.email_i)
+     cy.get('.btn').should('be.disabled')
+     cy.get('input[type=password]').type(userData.password1)
+     cy.get('button[type=submit]').click()
+     cy.contains(' email is invalid ').should('be.visible')
+    })
+  })
+
+  it('With out Username', () => {
+    cy.get('@userData').then((userData) => { 
+      cy.get('.form-control').eq(1).type(userData.email_two)
+      cy.get('input[type=password]').type(userData.password1)
+      cy.wait(3000)
+      cy.get('button[type=submit]').click()
+      expect(Cypress.env('MessageOne')).to.exist
```

Comment:
Some messages are defined in this file here (like `' email is invalid '`) and others in `cypress.json`.

I'd define all of them in the tests, to avoid having a giant `cypress.json` file, what do you think?

Reply:
If you need to use them in multiple places you could use constants in this file too.